backend/src/user/repository/user-typeorm.repository.ts

```diff
@@ -27,6 +27,13 @@ export class UserTypeOrmRepository implements UserRepository {
   async insert(user: User): Promise<string> {
     const newUser = this.userRepository.create({
       nickname: user.getNickname(),
+      token: user.getToken(),
```

Comment:
In the user model, create a function called toEntity that returns all fields to use like a object

like
```ts
class UserModel {
 private props: UserProps
 
 
 toEntity() {
  return this.props
 }
}

```